src/plugins/piral-breadcrumbs/README.md

```diff
@@ -73,6 +73,20 @@ export function setup(piral: PiletApi) {
 }
 ```
 
+### Rendering Breadcrumbs
+
+Within pilets, you typically _cannot_ access the `Breadcrumbs` component from the `piral-breadcrumbs`. While there are several options to work around this, the simplest is to use a built-in extension provided by Piral called `piral-breadcrumbs`:
+
+```tsx
+// Within any pilet:
+return (
+  <>
+    <Extension name="piral-breadcrumbs" />
```

Comment:
As `Extension` might not be known / defined (or another framework is used) I'd prefer to skip "return (..." and just have 

```html
<piral-extension name="piral-breadcrumbs"></piral-extension>
```

This is the generic web component and thus it can be used in any / most frameworks.